src/core/Form/SearchInput/SearchInput.md

```diff
@@ -4,5 +4,6 @@ import { SearchInput } from 'suomifi-ui-components';
 <SearchInput
   onBlur={event => console.log(event.target.value)}
   labelText="Search..."
+  labelMode="hidden"
```

Comment:
Example with and without?
Can wrap multiple components here with `<></>`

Reply:
Had that when testing myself. Probably a good idea to have them both.